experiments/long_runs/bucket.jl

```diff
@@ -106,19 +95,13 @@ domain =
 
 # Parameters
 params = LP.LandParameters(FT)
-σS_c = FT(0.2)
```

Comment:
I will check if any of these are behavior changing - we can adjust the toml file if needed then in a later PR if the long runs change much